Abilities/npc_dota_hero_troll_warlord/troll_warlord_whirling_axes_ranged.cs

```diff
@@ -0,0 +1,61 @@
+﻿// <copyright file="troll_warlord_whirling_axes_ranged.cs" company="Ensage">
+//    Copyright (c) 2017 Ensage.
+// </copyright>
+
+namespace Ensage.SDK.Abilities.npc_dota_hero_troll_warlord
+{
+    using Ensage.SDK.Abilities.Components;
+    using Ensage.SDK.Extensions;
+    using Ensage.SDK.Helpers;
+
+    public class troll_warlord_whirling_axes_ranged : ConeAbility, IHasTargetModifier
+    {
+        public troll_warlord_whirling_axes_ranged(Ability ability)
+            : base(ability)
+        {
+        }
+
```

Comment:
overwrite CanUse and check if correct melee/ranged form is currently active for troll abilities?

Reply:
okey